src/Microsoft.Framework.Runtime/PatternsCollectionHelper.cs

```diff
@@ -13,7 +13,12 @@ internal static class PatternsCollectionHelper
     {
         private static readonly char[] PatternSeparator = new[] { ';' };
 
-        public static IEnumerable<string> GetPatternsCollection(JsonObject rawProject, string projectDirectory, string projectFilePath, string propertyName, IEnumerable<string> defaultPatterns = null)
+        public static IEnumerable<string> GetPatternsCollection(JsonObject rawProject,
+                                                                string projectDirectory,
+                                                                string projectFilePath,
+                                                                string propertyName,
+                                                                IEnumerable<string> defaultPatterns = null,
+                                                                bool literalPath = false)
```

Comment:
There're a few places. Plus the parameters list is already long and in most cases it is not a literal path.